src/models/event-timeline-set.ts

```diff
@@ -56,6 +56,41 @@ export interface IRoomTimelineData {
     liveEvent?: boolean;
 }
 
+export interface IAddLiveEventOptions {
+    /** Applies to events in the timeline only. If this is 'replace' then if a
+     * duplicate is encountered, the event passed to this function will replace
+     * the existing event in the timeline. If this is not specified, or is
+     * 'ignore', then the event passed to this function will be ignored
+     * entirely, preserving the existing event in the timeline. Events are
+     * identical based on their event ID <b>only</b>. */
+    duplicateStrategy?: DuplicateStrategy;
+    /** Whether the sync response came from cache */
+    fromCache?: boolean;
+    /** The state events to reconcile metadata from */
+    roomState?: RoomState;
+    /** Whether the timeline was empty before the marker arrived in
+     *  the room. This could be happen in a variety of cases:
+     *  1. From the initial sync
+     *  2. It's the first state we're seeing after joining the room
+     *  3. Or whether it's coming from `syncFromCache` */
+    timelineWasEmpty?: boolean;
+}
+
+export interface IAddEventToTimelineOptions {
+    /** Whether to insert the new event at the start of the timeline */
+    toStartOfTimeline: boolean;
```

Comment:
`toStartOfTimeline` is ambiguous to what the start is. The timeline could be in chronological or reverse chronological order.

We should clarify whether this adds to the most recent side or the oldest side.

Reply:
It looks like a timeline is in chronological order with the most recent events being at the end.

`toStartOfTimeline` adds to the beginning of the list (index `0`) where the oldest events are.